NOTICE

```diff
@@ -0,0 +1,15 @@
+Puppet Module - puppetlabs-powershell
+
+Copyright 2013 - 2016 Puppet, Inc.
```

Comment:
Whoops - totally missed this - We'll need to find out from @joshcooper on what this should be.